beta/azure-container-apps/ReadMe.md

```diff
@@ -0,0 +1,228 @@
+
+## Prerequisites
+
+- A 1Password account with an active 1Password Business subscription or trial
+  > **Note**
+  >
+  > Try 1Password Business free for 14 days: <https://start.1password.com/sign-up/business>
+- An Azure account with permissions to create a Container App.
+  > **Note**
+  >
+  > If you don't have a Azure account, you can sign up for a free trial with starting credit: <https://azure.microsoft.com/en-us/free/>
+- [Azure CLI tool](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) with the ContainerApp Extension  or access to the [Azure Cloud Shell](https://learn.microsoft.com/en-us/azure/cloud-shell/quickstart) if performing the command driven deployment.
+
+## Getting started
+
+### Step 1: Generate credentials for automated user provisioning with 1Password
+
+1. [Sign in](https://start.1password.com) to your account on 1Password.com.
+2. Click [Integrations](https://start.1password.com/integrations/directory) in the sidebar.
+3. Choose your identity provider from the User Provisioning section.
+4. Choose "Custom deployment".
+5. Use the "Save in 1Password" buttons for both the `scimsession` file and `bearer token` to save them as items in your 1Password account. 
+6. Also use the download icon next to the `scimsession` file to save this file on your system.
+
+### Step 2: Configure the `scimession` credentials for passing to Container App
+
+The `scimsession` credentials will be saved as an secret variable in Container App. These credentials have to be Base64-encoded to pass them into the environment, but they're saved as a file in your 1Password item.
+
+1. Create the base64 value of your scimsession file: 
+
+   - On a **macOS** or **Linux** this can be done with: `cat <path>/scimsession | base64`. Copy the output value, it will be needed to create the secret for the deployment.
+   - On a **Windows** this can be done with: `[Convert]::ToBase64String([IO.File]::ReadAllBytes((Join-Path $PWD.Path 'scimsession')))`, copy the value it will be needed to create the secret for the deployment.
+
+2. Now that you have the base64 value copied, store this value in your scimsession 1Password item you saved earilier. 
+
+   - Locate the **scimsession file item** in your 1Password account.
+   - Click **edit**.
+   - Select **Add another field**.
+   - Select **Password**. 
+   - Paste the base64 value into the password field. 
+   - Rename the Password label above the value you just pasted, to something like `Base64 Value`. 
+   - Select **Save**.
+
+## Deploy 1Password SCIM Bridge to Azure Container App
+
+## Automatic Azure Deployment Steps using Azure CLI
+
+Using the Azure Cloud Shell can be easier (the assumption for the commands below is that you are using the Bash Cloud Shell), but this can also been done directly on yours system with the Azure CLI tool with the ContainerApp Extension
```

Comment:
Style ([easier](https://support.1password.com/style-guide/#ease)). Can we validate the below with PowerShell and note the differences in the items where there are any?
```suggestion
We recommend using the Azure Cloud Shell, but you can also deploy using the Azure CLI tool with the ContainerApp Extension. The example commands below assume you are using Bash:
```

Reply:
Updated to provide Powershell options for all steps for the Azure Cloud Shell. 
```suggestion
Using the Azure Cloud Shell can be easier, but this can also been done directly on yours system with the Azure CLI tool with the ContainerApp Extension
```